Incorrect tests: leading underscore in numeric literals

As reported by slang:

    ../..tests/generic/number/number_test_63.sv:7:25: error: numeric literals must not start with a leading underscore
    parameter int foo = 32'H_7f_FF_;
                            ^

This restriction is specified in LRM 5.7.1

I think number_test_63 might be the only one failing, though it's included in 3 sections. It's probably worth just changing this to be expect fails, since it could catch other tools that don't correctly implement this restriction.

--- a/tests/generic/number/number_test_63.sv
+++ b/tests/generic/number/number_test_63.sv
@@ -1,7 +1,7 @@
 /*
 :name: number_test_63
-:description: Test
-:should_fail: 0
+:description: Hex literal with invalid leading underscore prefix should fail.
+:should_fail: 1
 :tags: 5.6.4 5.7.1 5.7.2
 */
 parameter int foo = 32'H_7f_FF_;

